scipy/signal/_peak_finding.py

```diff
@@ -293,13 +308,13 @@ def peak_prominences(vector, peaks, wlen=None):
     Find all peaks and calculate prominences
 
     >>> from scipy import signal
-    >>> peaks = signal.argrelmax(vector)[0]
+    >>> peaks = signal.find_peaks(vector)
     >>> prominences = signal.peak_prominences(vector, peaks)
     >>> prominences
     array([ 1.24159486,  0.47840168,  0.28470524,  3.10716793,  0.284603  ,
             0.47822491,  2.48340261,  0.47822491])
 
-    Calculate the height of each peaks contour line
+    Calculate the height of each peak's contour line
 
     >>> contour_heights = vector[peaks] - prominences
```

Comment:
If not used you can combine this line to the group below.

Reply:
I don't quite understand. It is used in the call to `vlines` below. This is an extra line with comment to make the example as simple and explicit as possible.

Reply:
I meant if this is only going to be used in the plotting section below you don't need to make it as a separate code block. Now it is divided into three code blocks for no apparent reason. You can see the output here 

https://5174-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.signal.peak_prominences.html#scipy.signal.peak_prominences

You actually write what I meant in the `scipy.signal.find_peaks` docstring 😃